### PR TITLE
Update for the January Workshop

### PR DESCRIPTION
**Pull request by marcoow**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,8 +25,8 @@
   <div class="col-md-5 col-sm-7">
     <div class="row">
       <div class="col-sm-12">
-        <p>This workshop will <strong>take participants through building a full Ember.js application</strong> complete with a backend API and authentication, all the way to deployment in 3 days. The <strong>last day will also cover advanced topics</strong> like Animations and realtime data with Websockets.</p>
-        <p>While no previous Ember.js knowledge is required, you should be comfortable with things like JavaScript, jQuery, HTML, CSS and AJAX. So you can focus 100% on Ember, all assets, CSS and the API server will be provided - <strong>all you need is a laptop</strong>.</p>
+        <p>This 3 day workshop <strong>take participants through building a full Ember.js app</strong> using modern Ember.js patterns like <strong>Data down, Actions up, angle bracket components</strong> etc. The last day will also cover advanced topics like Animations and realtime data with Websockets.</p>
```
Comment (oliverbarnes) [inline index.html]: This 3 day workshop will*, or This 3 day workshop takes*

Comment (oliverbarnes) [inline index.html]: Should we add something about the last day's topics being open for requests?

Comment (marcoow) [inline index.html]: Good idea. Want to suggest sth.?

File: index.html [hunk 2]
```diff
@@ -25,8 +25,8 @@
   <div class="col-md-5 col-sm-7">
     <div class="row">
       <div class="col-sm-12">
-        <p>This workshop will <strong>take participants through building a full Ember.js application</strong> complete with a backend API and authentication, all the way to deployment in 3 days. The <strong>last day will also cover advanced topics</strong> like Animations and realtime data with Websockets.</p>
-        <p>While no previous Ember.js knowledge is required, you should be comfortable with things like JavaScript, jQuery, HTML, CSS and AJAX. So you can focus 100% on Ember, all assets, CSS and the API server will be provided - <strong>all you need is a laptop</strong>.</p>
+        <p>This 3 day workshop <strong>takes participants through building a full Ember.js app</strong> using modern Ember.js patterns like <strong>Data down, Actions up, angle bracket components</strong> etc. The last day will also cover advanced topics like Animations and realtime data with Websockets.</p>
```
Comment (LevelbossMike) [inline index.html]: We could tell people about the actual app that they will build in the workshop instead of the more generic 'full Ember.js app'

Comment (marcoow) [inline index.html]: I thought saying "music player app" might make people think they would specifically learn about ways to building music players only…

Comment (oliverbarnes) [inline index.html]: how about `...takes participants through building a full Ember.js app (a simple music player)`?

**Pull request merged**